### PR TITLE
Multiple route/outing fixes 

### LINKED ISSUE #528: Incorrect user names in some pages
In the following pages, user links use the user's login instead of his/her full name:
* [x] outing cards
* [x] associated user cards in the outing editing form
* [x] document history pages
* [x] document diff pages
* [ ] document version pages

OK:
* outing detail view pages

### PR DESCRIPTION
**Pull request by ginold**
related to #405 altitude de la neige a la descente
related to #403 fields alignment 
related to #398 orientations missing commas 
related to #546 avalanche signs

partly fixes #528 name instead of username

'name' is not returned by the API on diff and history pages so you will get an error.

### CODE REVIEW

File: c2corg_ui/templates/outing/edit.html [hunk 1]
```diff
@@ -367,14 +367,18 @@ <h2 class="heading show-phone"><span translate>conditions</span></h2>
...
-              <div class="input-group">
-                <select class="form-control" ng-model="outing.avalanche_signs" ng-options="rat as rat for rat in ${avalanche_signs}"><option value=""></option></select>
-                <span class="input-group-addon"><span class="glyphicon glyphicon glyphicon-alert"></span></span>
-              </div>
-              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="outing.avalanche_signs"></span>
+              <ul class="types">
+                <li ng-click="outing.avalanche_signs.indexOf('no') == -1 ? (outing.avalanche_signs = ['no']) : (outing.avalanche_signs[outing.avalanche_signs.indexOf('no')] = null)">
+                  <div class="checkbox"><input type="checkbox" ng-checked="outing.avalanche_signs.indexOf('no') > -1"><span>{{'no' | translate}}</span></div>
+                </li>
+                ## don't show other options if 'no' is selected
```
Comment (asaunier) [inline c2corg_ui/templates/outing/edit.html]: What if the user has checked other options before checking "no"? The other options will be submitted as well, won't they?

Comment (ginold) [inline c2corg_ui/templates/outing/edit.html]: They would, but it is managed to not do so in `formatOuting` function.

**Comment by asaunier** [comment]
Looks good to me. Please merge.

**Comment by tsauerwein** [comment]
> 'name' is not returned by the API on diff and history pages so you will get an error.

In the future please do no merge if you are knowingly breaking what worked before. It took me a moment to figure out why the history and version pages did not work anymore.

**Pull request merged**